src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -188,7 +194,7 @@ void CompositionalMultiphaseBase::registerDataOnMesh( Group & meshBodies )
       subRegion.registerExtrinsicData< dPhaseMobility_dGlobalCompDensity >( getName() ).
         reference().resizeDimension< 1, 2 >( m_numPhases, m_numComponents );
 
-      if( m_computeCFLNumbers )
+      if( m_computeStatistics )
```

Comment:
Does it make sense that `phaseOutflux`, `componentOutflux`, `phaseCFLNumber`, `componentCFLNumber` are conditionally register on `if( m_computeStatistics )`?
I could spot no usage of them related to the statistics display.

Reply:
Yes, I think it makes sense. Now, the CFL numbers are computed only when `computeStatistics` is true, in which case we have to register these quantities (`phaseOutflux`, `componentOutflux`, etc). If `computeStatistics` is `false`, the CFL numbers are not computed and these quantities are not needed.